runtime/interpreter/statictype.go

```diff
@@ -945,3 +945,39 @@ func (p TypeParameter) String() string {
 	}
 	return builder.String()
 }
+
+type CapabilityControllerStaticType struct {
+	BorrowType StaticType
```

Comment:
The type `CapabilityController` itself has no type parameters (at the moment). The borrow type of a capability controller is a run-time type, it is a member of the `CapabilityController` value.